### PR TITLE
Implement a workaround for relative Content Handlers

### PR DESCRIPTION
**Pull request by scottwallacesh**
Fixes, #311

### CODE REVIEW

File: gabbi/runner.py [hunk 1]
```diff
@@ -243,6 +253,13 @@ def _make_argparser():
         help='Custom response handler. Should be an import path of the '
              'form package.module or package.module:class.'
     )
+    parser.add_argument(
+        '-l', '--local-handlers',
+        default=False,
+        dest='local_handlers',
+        action='store_true',
+        help='Response handlers are relative to the current directory.'
```
Comment (cdent) [inline gabbi/runner.py:261]: are -> may be

As the change you've done is making it so the can be loaded locally, but they might also be loaded from somewhere else if they are not found on .

Comment (scottwallacesh) [reply gabbi/runner.py]: Updated.

**Comment by scottwallacesh** [comment]
Did you need me to squash the commits so they're clean?

**Comment by scottwallacesh** [comment]
Squashed!

**Comment by cdent** [comment]
I was just getting to merge this and thought of a corner case:

We need to prepend . not append. If we are testing a new version of a handler, and it is local to `.` but there is an already installed version in site-lib, appending . won't help.

Is that right?

Also, if you can think of a reasonable additional test, it could go in gabbi/tests/test_runner.py

**Comment by scottwallacesh** [comment]
Good thought.  Yes, we now prepend rather than append to the path.

**Pull request merged**